Packs/PrismaCloud/Integrations/PrismaCloudV2/PrismaCloudV2.py

```diff
@@ -200,17 +200,35 @@ def remediation_command_list_request(self, time_range: Dict[str, Any], alert_ids
     def alert_remediate_request(self, alert_id: str):
         self._http_request('PATCH', f'alert/remediation/{alert_id}', resp_type='response')
 
+    def remove_additional_resource_fields(self, input_dict, keys):
+        if isinstance(input_dict, dict):
+            return {k: self.remove_additional_resource_fields(v, keys) for k, v in input_dict.items() if k not in keys}
+        elif isinstance(input_dict, list):
+            return [self.remove_additional_resource_fields(element, keys) for element in input_dict]
+        else:
+            return input_dict
+
     def config_search_request(self, time_range: Dict[str, Any], query: str, limit: Optional[int] = None,
                               search_id: Optional[str] = None, sort_direction: Optional[str] = None,
-                              sort_field: Optional[str] = None):
+                              sort_field: Optional[str] = None, include_resource_json: Optional[str] = 'true',
+                              include_additional_resource_fields: Optional[str] = 'true',
+                              ):
         data = remove_empty_values({'id': search_id,
                                     'limit': limit,
                                     'query': query,
                                     'sort': [{'direction': sort_direction, 'field': sort_field}],
                                     'timeRange': time_range,
+                                    'withResourceJson': include_resource_json,
                                     })
 
-        return self._http_request('POST', 'search/config', json_data=data)
+        ret_value = self._http_request('POST', 'search/config', json_data=data)
+
+        if not include_additional_resource_fields:
+            keys_to_remove = ['shieldedInstanceInitialState', "configure-sh"]
+            demisto.debug(f'{include_additional_resource_fields=}, removing the fields {keys_to_remove}')
+            ret_value = self.remove_additional_resource_fields(ret_value, keys_to_remove)
```

Comment:
This should be handled as part of `config_search_command` and not `config_search_request`, to follow the logic in other functions (where the `_request` handles only the request itself, and the `_command` handles parsing the arguments and the response).

Reply:
[Done](https://github.com/demisto/content/pull/32606/commits/5fa21493d916ad6da2ccfe22910449fdbc49d6d5).